docs/section19/19-26.md

```diff
@@ -0,0 +1,99 @@
+        - Check if each value in the design_cooling_multiplier_schedule aligns with the misc_max_value: `for x in list(misc_design_cooling_multiplier_sch):`
+            - Check if the hourly value does not equal the highest hourly value used in the multiplier schedule: `if x != misc_max_value: misc_pass_cooling = false`  
+
+    - **Rule Assertion:** 
+    - Case 1: If the space type is not dwelling unit and all schedules pass per above then pass: `if is_dwelling_unit == false and all(inf_pass_heating, inf_pass_cooling,occ_pass_heating, occ_pass_cooling,int_lgt_pass_heating, int_lgt_pass_cooling,misc_pass_heating, misc_pass_cooling) == true: outcome = "PASS"`  
+    - Case 2: Else if the space type is not dwelling unit and one or more of the schedules does not pass per above then fail: `elif is_dwelling_unit == false and all(inf_pass_heating, inf_pass_cooling,occ_pass_heating, occ_pass_cooling,int_lgt_pass_heating, int_lgt_pass_cooling,misc_pass_heating, misc_pass_cooling) == false: outcome = "FAIL" and raise_message "<Insert space.id> does not appear to have followed this rule per Section G3.1.2.2.1 for one more more of the following heating or cooling design schedules: infiltration, occupants, lighting, gas and electricity using equipment"`  
+    - Case 3: Else if space type is dwelling unit and one or more of the design heating schedules fail per above then fail : `elif is_dwelling_unit == true and all(inf_pass_heating, occ_pass_heating, int_lgt_pass_heating, misc_pass_heating) == false: outcome = "FAIL" and raise_message "<Insert space.id> appears to be a dwelling unit and does not appear to have followed this rule per Section G3.1.2.2.1 for one more more of the following heating design schedules (cooling design schedules fall under the exception to Section G3.1.2.2.1 and were not assessed for dwelling units in this check): infiltration, occupants, lighting, gas and electricity using equipment"`  
+    - Case 4: Else if the space type is dwelling unit and all design heating schedules pass then the outcome is NOT_APPLICABLE (or UNDETERMINED it depends on our internal discussion with regard to assessing the exception as part of another RDS): `elif is_dwelling_unit == true and all(inf_pass_heating, occ_pass_heating, int_lgt_pass_heating, misc_pass_heating) == true: outcome = "NOT_APPLICABLE" and raise_message "<Insert space.id> appears to be a dwelling unit and cooling design schedules fall under the exception to Section G3.1.2.2.1 and were not assessed as part of this check. Note all heating design schedules passed this check for this space."`  
+
+
+**Notes/Questions:**  
+1. 90.1 definition of dwelling unit " a single unit providing complete independent living facilities for one or more persons, including permanent provisions for living, sleeping, eating, cooking, and sanitation." Not sure any other space types other than dwelling unit meet this definition. 
```

Comment:
Agreed. But I think we should handle multifamily building types differently from other building types for this check. E.g., for building types other than multifamily, assume that if space type is not specified that it's not a "dwelling unit"; for multifamily, if space type is not specified, than assume that it is a dwelling unit.

Reply:
CML need to update